config/mappings.go

```diff
@@ -456,13 +455,14 @@ func (m *MappingField) GetFields() []string {
 	return m.fields
 }
 
-func (m *MappingField) paserName() error {
+func (m *MappingField) parseName() error {
 	fieldName := ""
 	switch ch := m.s.Scan(); ch {
 	case '.':
 		return m.Parser()
 	case '[':
 		//Done
+		//TODO when is fieldName not nil?
```

Comment:
@lixingwang I noticed you check when fieldName isn't null, when is that the case, is there something missing here?

Reply:
@fm-tibco seems no case the fieldName is null, please remove the code.